INFO 2024-07-30 14:35:43,815 the number entered by user is 2 and 3
INFO 2024-07-30 14:42:34,745 the number entered by user is 2 and 3
INFO 2024-07-30 14:42:34,745 we are into function
INFO 2024-07-30 14:42:34,745 we have competed division operation
INFO 2024-07-30 14:42:45,426 the number entered by user is 2 and 0
INFO 2024-07-30 14:42:45,427 we are into function
ERROR 2024-07-30 14:42:45,427 division by zero
Traceback (most recent call last):
  File "C:\Users\alekh\PycharmProjects\oPythonClasses\Logging\logging3.py", line 9, in divide
    div = a/b
          ~^~
ZeroDivisionError: division by zero
INFO 2024-07-30 14:47:12,212 the number entered by user is 2 and 0
INFO 2024-07-30 14:47:12,212 we are into function
ERROR 2024-07-30 14:47:12,212 division by zero
Traceback (most recent call last):
  File "C:\Users\alekh\PycharmProjects\oPythonClasses\Logging\logging3.py", line 9, in divide
    div = a/b
          ~^~
ZeroDivisionError: division by zero
INFO 2024-07-30 14:47:28,408 the number entered by user is 2 and 5
INFO 2024-07-30 14:47:28,408 we are into function
INFO 2024-07-30 14:47:28,408 we have competed division operation
INFO 2024-07-30 14:47:28,408 the result is 0.4
INFO 2024-07-30 14:52:43,257 the number entered by user is 2 and 5
INFO 2024-07-30 14:52:43,258 we are into function
INFO 2024-07-30 14:52:43,258 we have competed division operation
INFO 2024-07-30 14:52:43,258 the result is 0.4
